mysql-test/suite/innodb/r/percona_encrypt_crash.result

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE  t1 ( c0 INT(66) AUTO_INCREMENT,  c1 CHAR(30),  c2 INT,  c3 INT,  c4 INT,  c5 INT,  c6 INT, INDEX tt_10_pi0(c2, c1 DESC, c3, c0 ASC), INDEX tt_10_pi1(c0, c2 ASC, c1), INDEX tt_10_pi2(c2, c3 ASC, c1 DESC, c0 ASC) ) ENCRYPTION='Y' ROW_FORMAT=COMPRESSED ENGINE=INNODB;
```

Comment:
Please add a comment with PS-5791 : crash recovery bug (...). We may add more bugs to this testcase.